### ISSUE TITLE
Fetch a list of all table numbers from all databases

### ISSUE THREAD

**Issue by pmayd** [CorrelAid/pystatis#29]

**Comment by PiaSchroeder** [comment]
I have the lists (based on catalogue endpoints) but I'm not sure where to put them (the wiki only allows text?).

**Comment by pmayd** [comment]
Oh wow cool you got the paging working by yourself? For now we don't need the lists but you can put them in the repo, they are not part of the package unless I say so in the pyproject.toml so feel free to add a new folder like reference_data

**Comment by PiaSchroeder** [comment]
Not really, I used the catalogue/statistics endpoint to get a list of all statistics and then I used the statistics codes to filter the tables in the catalogue/tables requests to downsize them. It's not ideal but I haven't found a way of requesting specific pages. I sent an email to the genesis people though to be sure, should there be any tables I've missed with this method, I'll update the files.

**Comment by pmayd** [comment]
I see page length in the query can be up to 2500, did you try this?

**Comment by PiaSchroeder** [comment]
yes but if I do that it takes forever and then crashes with connection aborted errors or internal server errors. Also, genesis-online has more than 2500 tables (which is the max page length) and I don't see a way to request specific pages (as in not starting from page 1)..

**Comment by PiaSchroeder** [comment]
[Notebook retrieving all stats and tables](https://github.com/PiaSchroeder/pystatis_aux/blob/main/get_stats_and_tables.ipynb)

**Comment by pmayd** [comment]
Notebook looks great!
